### PR TITLE
Tdr 131 add av metadata

### PR DESCRIPTION
**Pull request by TomJKing**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,29 @@ These classes are used by other TDR repositories to communicate with the consign
...
+This will place a snapshot version of the built project jar in the local .ivy cache folder: $HOME/.ivy2/local/uk.gov.nationalarchives/tdr-generated-graphql_2.13/[version number]-SNAPSHOT
+
+Other sbt projects that have this project as a dependency can access the local snapshot version by changing the version number in their build.sbt file, for example:
+  ```
+  ... other dependencies...
+  "uk.gov.nationalarchives" %% "tdr-generate-graphql" % "[version number]-SNAPSHOT"
+  ... other dependences...
+  ```
+  
+#### Typescript version
```
Comment (suzannehamilton) [inline README.md:24]: 👍

File: src/main/graphql/AddAVMetadata.graphql [hunk 2]
```diff
@@ -0,0 +1,11 @@
+mutation AddAVMetadata($input: [AddAVMetadataInput!]!) {
+    addAVMetadata(addAVMetadataInput: $input) {
+        fileId
+        software
+        value
+        softwareVersion
+        databaseVersion
```
Comment (suzannehamilton) [inline src/main/graphql/AddAVMetadata.graphql:7]: Did we decide all these fields were necessary? I think Sam knew about what fields were appropriate for YARA.

Comment (TomJKing) [reply src/main/graphql/AddAVMetadata.graphql:7]: I would rather leave all fields in as it is still vague what is required for yara. If when yara is set up its found that some fields are not needed they can be removed.

Comment (suzannehamilton) [reply src/main/graphql/AddAVMetadata.graphql:7]: OK, so we'll come back to it when we do https://national-archives.atlassian.net/browse/TDR-129?

**Pull request merged**